nimterop.nimble

```diff
@@ -46,15 +46,37 @@ proc testAll() =
 
 const htmldocsDir = "build/htmldocs"
 
+when (NimMajor, NimMinor, NimPatch) >= (0, 19, 9):
+  import os
+  proc getNimRootDir(): string =
+    #[
+    hack, but works
+    alternatively (but more complex), use (from a nim file, not nims otherwise
+    you get Error: ambiguous call; both system.fileExists):
+    import "$nim/testament/lib/stdtest/specialpaths.nim"
+    nimRootDir
+    ]#
+    fmt"{currentSourcePath}".parentDir.parentDir.parentDir
+
```

Comment:
How about putting this in compat and importing? Not sure if nimble can import but worth checking.

Reply:
it can't before I push https://github.com/nimterop/nimterop/pull/113 (upgrade to 0.19.4) so lemme address this in a follow up PR